omod/src/main/webapp/pages/results.gsp

```diff
@@ -156,7 +218,14 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
         patients.push("${prettify(ui.encodeHtmlContent(it.name))}");
     </script>
     <tr>
-        <td style="display: none">rule_${it.rule.ruleId}</td>
+        <td class="hidden">rule_${it.rule.ruleId}</td>
+
+        <% if (it.patientProgram) { %>
```

Comment:
@shivtej1505 The checks here should be inside the td since all that is happening is changing what content is displayed